doc/spec.md

```diff
@@ -3505,11 +3505,14 @@ See also: `string·elems`.
 <a id='string·capitalize'></a>
 ### string·capitalize
 
-`S.capitalize()` returns a copy of string S with all Unicode letters
-that begin words changed to their title case.
+`S.capitalize()` returns a copy of string S with its first code point
```

Comment:
Python says "first character" instead of "first code point". Is the difference intended?

In Python:

```
>>> s = '\uFB03'
>>> s
'ﬃ'
>>> s.capitalize()
'FFI'
>>> len(s.capitalize())
3
```

Reply:
Sorry, this comment was too nitpicky. I don't think the Python behavior actually makes sense: given the wording, I'd expect it to return three characters: F f i.

I think having the spec say this applies to code points, not characters makes sense. It may be good to point out somewhere that this is different than Python though.

Reply:
The Python function is even more underspecified than ours: it seems to attempt to uppercase (not capitalize) the decomposition of a lower case letter that has no corresponding uppercase letter. I don't want to go there.

Thanks for educating me about this particular naughty string. :)